package.json

```diff
@@ -1,9 +1,9 @@
 {
   "name": "app-name",
   "private": true,
-  "dependencies": {},
   "version": "0.1.0",
   "devDependencies": {
+    "@lavamoat/preinstall-always-fail": "2.1.1",
```

Comment:
still need this one? I would have thought the CI would fail based on this name. Maybe am confused tho lol

Reply:
oh I see, its a failsafe ?

Reply:
correct, it does 1 thing runs a script on post install that forces the yarn install to fail, so if we have screwed up the config that blocks post install scripts itll block CI